stdlib/public/core/Optional.swift

```diff
@@ -409,6 +409,28 @@ extension Optional : Equatable where Wrapped : Equatable {
   }
 }
 
+extension Optional : Hashable where Wrapped : Hashable {
+    /// The hash value for the optional.
```

Comment:
Nit: this file, like others in the standard library, uses two spaces to indent instead of four.